company-clang.el

```diff
@@ -80,6 +80,76 @@ or automatically through a custom `company-clang-prefix-guesser'."
     (insert-file-contents-literally file nil beg end)
     (buffer-string)))
 
+(defcustom company-clang-parse-documentation nil
+  "When non-nil, clang will parse completion's comments/documentation."
```

Comment:
Also, the docstring here should mention the minimum version Clang that supports this.